draft-ietf-quic-recovery.md

```diff
@@ -1114,7 +1117,8 @@ SetLossDetectionTimer():
     loss_detection_timer.update(loss_time)
     return
 
-  if (crypto packets are in flight):
+  if (crypto packets are in flight &&
+      crypto data to send):
```

Comment:
what does this predicate mean? That there's new data to send?

Reply:
It's supposed to mean that there's unacknowledged crypto data.  Should I say "unacknowledged crypto data" instead?

Reply:
I removed this from here due to the other PR, but left it in below.  Feel free to suggest alternate phrasing.